tools/sapp/sapp/tests/interactive_test.py

```diff
@@ -1600,6 +1600,8 @@ def testCreateIssueOutputStringNoSourcesNoSinks(self):
             code=1000,
             callable="module.function1",
             message="root",
+            min_trace_length_to_sources=1,
+            min_trace_length_to_sinks=1,
```

Comment:
Could also be worth adding an assert for `Min Trace Length ...` in this testing function

Reply:
Yes, it's worth adding test. I think it would be better to have a separate test function just for the `Min Trace Length ...`. I'll update the test code.